mk/xe-linux-distribution

```diff
@@ -147,6 +147,7 @@ identify_redhat()
                -e 's/^CentOS release \([0-9]*\)\.\([0-9]*\) (.*)/distro=centos;major=\1;minor=\2/gp;' \
                -e 's/^CentOS release \([0-9]*\) (.*)/distro=centos;major=\1/gp;' \
                -e 's/^CentOS Linux release \([0-9]*\)\.\([0-9]*\)\(\.[0-9]*\)\? (.*)/distro=centos;major=\1;minor=\2/gp;' \
+               -e 's/^Sangoma Linux release \([0-9]*\)\.\([0-9]*\)\(\.[0-9]*\)\? (.*)/distro=centos;major=\1;minor=\2/gp;' \
```

Comment:
I guess for `alphabetical order`, the `Sangoma` should stay with the `S*` one?
https://github.com/xenserver/xe-guest-utilities/blob/61b51c792ecd7cdab9b2c8a29143f0f6c880ad64/mk/xe-linux-distribution#L154

Reply:
Ah yes, updated :)